variables.tf

```diff
@@ -16,7 +16,7 @@ variable "region" {
 }
 
 variable "instance_count" {
-  description = "Number of EC2 instances to create."
+  description = "Number of ECS task to create."
```

Comment:
task -> tasks?
And if it's "tasks" isn't `instance_count` misleading? Or does a task == an instance?

Reply:
`instance_count` variable was named when we were using EC2 for WD.
Now we use ECS tasks and this variable defines the number of tasks we want to have.

I am updating `task` to `tasks` in other PR. but it will be difficult to rename the variable name.